lib/standards/aria-roles.js

```diff
@@ -340,7 +343,7 @@ const ariaRoles = {
     superclassRole: ['landmark']
   },
   marquee: {
-    type: 'widget',
+    type: 'structure',
```

Comment:
When you're adding roles to `valid-scrollable-semantics`, I think you can leave this one out. `role=marquee` doesn't seem like it should be focusable.